assets/helpers/payPalContributionsCheckout/__tests__/__snapshots__/payPalContributionsCheckoutReducerTests.js.snap

```diff
@@ -0,0 +1,17 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
```

Comment:
The snapshots files are autogenerated by Jest for [snapshot testing](https://facebook.github.io/jest/docs/snapshot-testing.html)